Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +17,30 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
+            int intTest = Lesson3.GetInt(message);
             // To do: call your implementation. 
-            return 0;
+            return intTest;
         }
 
         public static string PromptString(string message)
         {
+            string string1 = Lesson3.GetString(message);
```

Comment:
Since there is no other string in this context, it should be just string. However, naming variables after types is no good, therefore you should make it more human-understable. You read text- it should be text.

Reply:
I named it string1 because string string = threw a compiler error. But yeah, that makes a lot more sense